besu/src/test/java/org/hyperledger/besu/controller/ConsensusScheduleBesuControllerBuilderTest.java

```diff
@@ -98,4 +105,43 @@ public void mustCreateCombinedProtocolScheduleUsingProtocolSchedulesOrderedByBlo
     Mockito.verify(combinedProtocolScheduleFactory)
         .apply(expectedProtocolSchedulesSpecs, Optional.of(BigInteger.TEN));
   }
+
+  @Test
+  public void mustCreateSchedulableContext() {
+    final ConsensusContext context1 = Mockito.mock(ConsensusContext.class);
+    final ConsensusContext context2 = Mockito.mock(ConsensusContext.class);
+
+    final Map<Long, BesuControllerBuilder> besuControllerBuilderSchedule = new TreeMap<>();
+    besuControllerBuilderSchedule.put(0L, besuControllerBuilder1);
+    besuControllerBuilderSchedule.put(10L, besuControllerBuilder2);
```

Comment:
Is it worth using a HashMap or something and making these out of order to test the `.sorted(ForkSpec.COMPATOR)`?

Reply:
Having investigated doing this for https://github.com/hyperledger/besu/pull/3097#discussion_r762392580
I don't think it's possible due to the way ForksSchedule is implemented